src/vaadin-board-row.html

```diff
@@ -228,9 +228,13 @@
         _recalculateFlexBasis(forceResize) {
           const width = this.getBoundingClientRect().width;
           const breakpoints = this._measureBreakpointsInPx();
-          if (forceResize || width != this._oldWidth
-              || breakpoints.smallSize != this._oldBreakpoints.smallSize
-              || breakpoints.mediumSize != this._oldBreakpoints.mediumSize) {
+          const shouldRecalculate =
+            forceResize ||
+            (this.offsetParent !== null &&
+              (width != this._oldWidth ||
+                breakpoints.smallSize != this._oldBreakpoints.smallSize ||
+                breakpoints.mediumSize != this._oldBreakpoints.mediumSize));
+          if (shouldRecalculate) {
```

Comment:
Consider extracting this to separate method so that it could be used like this:

```js
if (forceResize || this._shouldRecalculate()) {
  // Do stuff
}
```